eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -3422,16 +3431,15 @@ class NormalStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with
     awaitCond(alice.stateName == CLOSING)
 
     val currentBlockHeight = alice.underlyingActor.nodeParams.currentBlockHeight
-    val blockTargets = alice.underlyingActor.nodeParams.onChainFeeConf.feeTargets
     if (commitFeeBumpDisabled) {
       val claimMain = alice2blockchain.expectMsgType[PublishFinalTx]
       assert(alice2blockchain.expectMsgType[WatchTxConfirmed].txId === aliceCommitTx.txid)
       assert(alice2blockchain.expectMsgType[WatchTxConfirmed].txId === claimMain.tx.txid)
       alice2blockchain.expectNoMessage(1 second)
     } else {
       val localAnchor = alice2blockchain.expectMsgType[PublishReplaceableTx]
-      // When there are no pending HTLCs, there is no rush to get the commit tx confirmed
-      assert(localAnchor.txInfo.confirmBefore === currentBlockHeight + blockTargets.commitmentWithoutHtlcsBlockTarget)
+      // When there are no pending HTLCs, there is no absolute deadline to get the commit tx confirmed, we use priority
+      assert(localAnchor.txInfo.confirmationTarget.isInstanceOf[ConfirmationTarget.Priority])
```

Comment:
nit: we can even be more accurate:

```suggestion
      assert(localAnchor.txInfo.confirmationTarget == ConfirmationTarget.Priority(ConfirmationPriority.Medium))
```

Reply:
Done in dbf895211.